hw_3-Async-development/.gitignore

```diff
@@ -0,0 +1,2 @@
+.vscode
+node_modules
```

Comment:
file gitignore is on;y necessary for the root repository, not for every internal folder